src/NHibernate/Dialect/MySQL8Dialect.cs

```diff
@@ -0,0 +1,9 @@
+using System.Data;
+
+namespace NHibernate.Dialect
+{
+	public class MySQL8Dialect : MySQL57Dialect
+	{
+		public MySQL8Dialect() => RegisterColumnType(DbType.Boolean, "BOOLEAN");
```

Comment:
This `BOOLEAN` type does not actually exist for MySql.

See [11.9 Using Data Types from Other Database Engines](https://dev.mysql.com/doc/refman/8.0/en/other-vendor-data-types.html), MySql goes on using `TINYINT` for this.

See also [Numeric Data Type Syntax](https://dev.mysql.com/doc/refman/8.0/en/numeric-type-syntax.html)
> `BOOL`, `BOOLEAN`
>
>These types are synonyms for [`TINYINT(1)`](https://dev.mysql.com/doc/refman/8.0/en/integer-types.html).

So, that is mere synonyms. And if that documentation is accurate, switching to them does not cease using the deprecated feature, display width.

Reply:
So, it should be:
```suggestion
		public MySQL8Dialect() => RegisterColumnType(DbType.Boolean, "TINYINT");
```